gear/gear/clients.py

```diff
@@ -18,6 +21,8 @@ def get_cloud_async_fs() -> AsyncFS:
     cloud = get_global_config()['cloud']
 
     if cloud == 'azure':
+        if os.environ.get('HAIL_TERRA'):
```

Comment:
Should we just use the presence of the Terra workspace env vars as evidence that we're in azure terra?

Reply:
ya we can do that.